### DISCUSSION TITLE
is this safe?

### DISCUSSION THREAD

**Discussion by SmdLeaf** [RedTeamPentesting/pretender#4] [Q&A] [answered]
is this spoofer safe to use and dose it work?

**Comment by rtpt-erikgeiser** [comment, marked as answer]
Of course `pretender` works. What exactly do you mean by "safe to use"? To safely use `pretender`, you should know what you are doing as it affects other computers on you network, but that's  the whole point of it. You can read [our blog post](https://blog.redteam-pentesting.de/2022/introducing-pretender/) in order to understand the different attack modes.
